packages/apollo-server-cloudflare/tsconfig.test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.test.base",
+  "include": ["src/**/__tests__", "src/**/__mocks__"],
+  "references": [
+    { "path": "./tsconfig.json" },
+    { "path": "../apollo-server-integration-testsuite" }
```

Comment:
Still not quite understanding the occasional explicit reference to `tsconfig.json` and occasionally referencing the root path (which I believe then implies `tsconfig.json`?

Reply:
Yep, it probably makes sense to be consistent here and drop it everywhere.